Synergism.js

```diff
@@ -1,6 +1,10 @@
-
-
-// BETA SAVE FUNCTION
+const intervalHold = [];
+const interval = new Proxy(setInterval, {

```

Comment:
Confused what proxy does exactly

Reply:
Proxy (as used in this case) is basically intercepting the setInterval method and allowing us to "add" onto it by pushing the interval's id to an array. 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Proxy

It's used here because it's the smallest and easiest solution. It is also completely backwards-compatible with setInterval - as shown in the commits all that is needed it to rename the method.